relevanceai/operations/cluster/cluster.py

```diff
@@ -13,6 +13,7 @@ def __init__(
         api_key: str,
         firebase_uid: str,
         model: str,
+        vector_fields: Optional[List[str]] = None,
         alias: Optional[str] = None,
         n_clusters: Optional[int] = None,
         config: Optional[Dict[str, Any]] = None,
```

Comment:
`config` should really be `cluster_args`

Reply:
I don't think so, but you're right that confit should be more specific. What about 'model_config'

Reply:
The issue with `model` instead of `cluster` is that if the model becomes a complex 2 object model (such as GAN clustering or something), it can lose meaning. Instead of  `model_config`, how about `cluster_config` ?